server/utils/conditionsProvider.ts

```diff
@@ -0,0 +1,14 @@
+import conditionsConfig from '../config/conditions'
+
+export default function getGroupedAdditionalConditions(): Record<string, unknown>[] {
+  const map = new Map()
+  conditionsConfig.additionalConditions.forEach(condition => {
+    const collection = map.get(condition.groupName)
+    if (!collection) {
+      map.set(condition.groupName, [condition])
+    } else {
+      collection.push(condition)
+    }
+  })
```

Comment:
Organising conditions into groups for display together?

Reply:
Yea, the template iterates through the groups and displays each condition within that group. This function collects the conditions into groups by looking at the `groupName` tags of each condition